src/transactions/OfferExchange.cpp

```diff
@@ -1281,6 +1282,68 @@ exchangeWithPool(int64_t reservesToPool, int64_t maxSendToPool, int64_t& toPool,
     }
 }
 
+static PoolID
+getPoolID(Asset const& x, Asset const& y, int32_t feeBps)
```

Comment:
Question that originated from @ire-and-curses : I looked in our codebase and we don't have any test vectors about "pool IDs".

As this function (that computes a pool ID given all parameters) is basically what we want, we should just add a couple test vectors to our code base that show that `getPoolID(cur1, cur2, LIQUIDITY_POOL_FEE_V18) == getPoolID(cur1, cur2, LIQUIDITY_POOL_FEE_V18) == 0xSOMEHASH`

Reply:
@ire-and-curses @paulbellamy I added a bunch of tests for pool IDs. You can find them in "Add tests for getPoolID" (currently 5e8d7cf but that's subject to change).